src/app/scripts/cac/search/cac-typeahead.js

```diff
@@ -128,7 +137,7 @@ CAC.Search.Typeahead = (function ($) {
                 '-75.126531',
                 '39.967842'
             ].join(','),
-            category: 'Address,POI',
+            category: searchCategories,
```

Comment:
I think these fields are also used on the geocoder. Do you know if they need to change there as well?